R/pkg/R/DataFrame.R

```diff
@@ -303,8 +303,28 @@ setMethod("colnames",
 #' @rdname columns
 #' @name colnames<-
 setMethod("colnames<-",
-          signature(x = "DataFrame", value = "character"),
```

Comment:
I would prefer keeping the current signature instead of checking the class of the parameter inside the method.
@felixcheung, what's your opinion?